### PR TITLE
feature/EUI-3911

### PR DESCRIPTION
**Pull request by rayliangatos**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3911
https://tools.hmcts.net/jira/browse/EUI-4103

### Change description ###
Reallocate role E2E journey for Judicial and LegalOps

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
@rayliangatos Reallocate and Remove Allocate Journey

File: src/role-access/components/case-roles-table/case-roles-table.component.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import { Component, ElementRef, Input, ViewChild } from '@angular/core';
 import { CaseView } from '@hmcts/ccd-case-ui-toolkit';
-import { UserType } from 'api/user/interfaces/user-type';
-import { Action } from '../../../role-access/models/case-role.interface';
+import { UserType } from 'src/cases/models/user-type';
```
Comment (connorpgpmcelroy) [inline src/role-access/components/case-roles-table/case-roles-table.component.ts:3]: src

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Just the one src on import that needs to be changed. Happy to approve after that's been changed

**Review by connorpgpmcelroy — APPROVED** [review]
Discussed concern with Ray regarding previous ticket which was addressed. Previous comment also addressed

**Pull request merged**